src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx

```diff
@@ -5,7 +5,8 @@ import { LambdaConfig } from '../../../utils/integrations';
 import { IntegrationTextInputField } from './IntegrationTextInputField';
 
 const Placeholders: LambdaConfig = {
-  role_arn: '<my lambda role ARN>',
+  role_arn: 'arn:aws:iam::123:role/lambda-function-role-arn',
+  exec_state: '',
```

Comment:
What is this even for? I was a little confused when reading this.

Reply:
We require they provide a role with permissions for us to create Lambda functions.

Reply:
@hsubbaraj-spiral Can you explain what we need the exec_state field here for?

Reply:
Yeah I meant the exec_state.

Reply:
I believe `exec_state` was added when we switched to connecting with Lambda asynchronously. VSCode threw an error saying the placeholder was missing a value so I added an empty placeholder for it. 

@Fanjia-Yan do we need the exec_state as part of the UI LambdaConfig? If only the backend uses the `exec_state`, we can make that field optional and remove it from the UI.

Reply:
The exec_state is to update the status of Lambda Connection on UI so I think the `exec_state` should be there at least when we receive the "lambda_config" from server to UI. 

I think we can either hardcoded the input to lambda config exec_state as null or we can create a new something like `LambdaExecStateConfig` to receive the output from server that contains the `exec_state`?

Reply:
I think it's fine to leave it as is, since we are using the value on the UI. It'll be fine to set the value to empty string on the UI at connection time.

Reply:
I can get rid of this as part of my integration refactor. All integrations now have an exec_state on their config, so we don't need a specialized field anymore.